src/metadata.rs

```diff
@@ -12,6 +12,8 @@ pub struct Metadata {
     title: String,
     season: Option<i32>,
     episode: Option<i32>,
+    last_episode: Option<i32>,
+    multi_episode: Option<Vec<i32>>,
```

Comment:
I don't think storing it twice is required, we can easily figure out the last episode from the Vec.

Reply:
I was thinking last_episode could go away. Now that there is the `Vec`

Reply:
last_episode removed
multi_episode renamed to `episodes` as it makes more sense and reads more naturally.